pipelines/ApVerifyWithFakes.yaml

```diff
@@ -14,13 +14,16 @@ tasks:
     config:
         connections.coaddName: parameters.coaddName
   # Parallel to ApPipe's imageDifferenceWithFakes, allows clean differencing metrics.
-  imageDifference:
-    class: lsst.pipe.tasks.imageDifference.ImageDifferenceFromTemplateTask
+  subtractImages:
+    class: lsst.ip.diffim.subtractImages.AlardLuptonSubtractTask
+    config:
+      connections.coaddName: parameters.coaddName
+      doApplyFinalizedPsf: False
```

Comment:
Is this line in case `ApVerifyWithFakes` some day uses a different subtracter from `ApPipeWithFakes`, preventing config inheritance? If so, a comment would be helpful.

Reply:
No, this change is unneeded. I'll remove it.